.github/CODEOWNERS

```diff
@@ -4,7 +4,7 @@
 # For syntax help see:
 # https://help.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-syntax
 
-*                       @googleapis/yoshi-java
+*                       @googleapis/api-logging @googleapis/yoshi-java
```

Comment:
This file is managed by templates. Please set the codeowner_team in the .repo-metadata.json file or ignore this file from the templates in synth.py